extensions/espanso/src/index.tsx

```diff
@@ -21,9 +22,22 @@ export default function Command() {
   useEffect(() => {
     async function fetchData() {
       try {
-        const { stdout: result } = await $`espanso match list -j`;
-        let matches: EspansoMatch[] = JSON.parse(result);
-        matches = matches.sort((a, b) => a.triggers[0].localeCompare(b.triggers[0]));
+        const { packages: packageFilesDirectory, match: matchFilesDirectory } = await getEspansoConfig();
+        const packageMatches = getMatches(packageFilesDirectory, { packagePath: true });
+        const userMatches = getMatches(matchFilesDirectory);
+        const matches: NormalizedEspansoMatch[] = userMatches.concat(packageMatches);
+        matches.sort((a, b) => {
```

Comment:
Export this function please 😌

Reply:
Could you say a little more about what you mean here as well?

Reply:
Yeah sorry, I was a bit busy haha.

It's actually a generic function, it could be extracted from the main function and being on the top or in utils.ts.

Reply:
Got it, will do.